src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -229,16 +229,14 @@ def pre_remove(self, daemon_id: str) -> None:
         """
         pass
 
-    def get_auth_entity(self, daemon_id: str, host: str = "") -> AuthEntity:
+    def get_auth_entity(self, daemon_id: str) -> AuthEntity:
         """
         Map the daemon id to a cephx keyring entity name
         """
         if self.TYPE in ['rgw', 'rbd-mirror', 'nfs', "iscsi"]:
             return AuthEntity('client.' + self.TYPE + "." + daemon_id)
         elif self.TYPE == 'crash':
-            if host == "":
-                raise OrchestratorError("Host not provided to generate <crash> auth entity name")
-            return AuthEntity('client.' + self.TYPE + "." + host)
+            return AuthEntity('client.' + self.TYPE + "." + daemon_id)
```

Comment:
This reverts #35274 by @jmolmo . please be careful here.

Reply:
reverted this PR back to the behavior seen in  #35274